### ISSUE TITLE
Screen reader is not announcing control name present in top menu navigation and left navigation on hovering with mouse in mouse tracking on mode:A11y_Visual Studio Code Client_Home screen_ScreenReader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#229478]
## GitHub Tags:
#A11yTCS; #A11ySev4; #Visual Studio Code Client; #BM_Visual Studio Code Client_Win32_JULY2024; #DesktopApp; #FTP; #A11yUsable; #A11yUsablehigh; #NVDA; #Screen reader; #Win32; #Benchmark;
## Environment and OS details:
Application Name: Visual Studio Code Client
OS: Windows 11 version 23H2 OS built: 22631.4169.
Screen reader: NVDA: version 2024.2
## Reproduction Steps:

1. Turning on the NVDA.
2. Open Visual studio code insider editor.
3. Turn on mouse tracking mode with "Insert + M" key.
4. Now hover the control present in top menu navigation and left navigation.
5. Observed that NVDA is not announcing the control name.

## Actual Result:
Screen reader is not announcing control name present in top menu navigation and left navigation on hovering with mouse in mouse tracking on mode.
## Expected Result:
Screen reader should announce the control name present in top menu navigation and left navigation on hovering with mouse in mouse tracking on mode.
## User Impact:
Screen reader user will not now the control name on mouse hover because screen reader is not announcing control name present in top menu navigation and left navigation on hovering with mouse in mouse tracking on mode.
## Attachments:

**Comment by kapilvaishna** [comment]
eDAD bug: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/229315

**Comment by meganrogge** [comment]
Your gif did not get attached fyi

**Comment by meganrogge** [comment]
Also, does this work in other apps? If so, can you provide an example?

**Comment by meganrogge** [comment]
A sev4 seems pretty high for an issue that no users have reported.

**Comment by kapilvaishna** [comment]
@meganrogge it works for visual studio client please find attachment below. and Sev4 is lowest severity we use, and it don't have any define SLA 

https://github.com/user-attachments/assets/d0f87041-d826-4bd5-822c-e742c3aed6cc
